vllm/config/cache.py

```diff
@@ -162,6 +174,15 @@ def _validate_cache_dtype(cls, cache_dtype: CacheDType) -> CacheDType:
             )
         return cache_dtype
 
+    @field_validator("pinned_prefix_cap_ratio", mode="after")
+    @classmethod
+    def _validate_pinned_prefix_cap_ratio(cls, value: float) -> float:
+        if value < 0.0 or value > 1.0:
+            raise ValueError(
+                f"pinned_prefix_cap_ratio must be within [0.0, 1.0], got {value}."
+            )
+        return value
```

Comment:
If you remake this PR, you don't need to write this manually, you can use `pydantic.Field(default=0.2, ge=0, le=1)`